amethyst_renderer/src/pipe/effect/mod.rs

```diff
@@ -100,7 +100,7 @@ impl Effect {
     }
 
     /// FIXME: Update raw buffer without transmute, use `Result` somehow.
```

Comment:
I don't know if its important, but it seems like a potential optimisation

Reply:
How exactly this is optimisation?

Reply:
I didn't knew what transmute was, I just assume from the way the comment was written that it was a bad thing. :)

Reply:
@jojolepro It's extremely dangerous to use, but if used correctly it can be quite helpful.